src/components/ConversionFormFromViewer/style.css

```diff
@@ -0,0 +1,106 @@
+.icon {
+    top: 15px;
+    right: 15px;
+    position: absolute;
+}
+
+.form-div :global(.ant-input) {
+    width: 200px;
+    height: 30px;
+}
+
+.number-input {
+    width: 75px !important;
+}
+
+.form-div :global(.ant-select) {
+    width: 200px;
+}
+
+:global(.ant-collapse) {
+    background-color: var(--overlay-purple);
+}
+.ant-collapse-content > .ant-collapse-content-box {
+    padding: 0px;
+}
+
+:global(.ant-form-item) {
+    padding: 3 3 3 3;
+}
+
+:global(.ant-form-horizontal) {
+    padding: 3 3 3 3;
+}
+
+input,
+select {
```

Comment:
don't use selectors like this. Should always be using class selectors to wrap elements if you need to style base elements